### PR TITLE
style: adjust variables and styles for Nc30

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* First batch of design adjustments with new styles

#### Admin settings

🏚️ Before | 🏡 After
-- | --
image | image
image | image

#### Chat

🏚️ Before | 🏡 After
-- | --
image | image
image | image
image | image
image | image

#### Call

🏚️ Before | 🏡 After
-- | --
image | image
image | image
image | image
image |image

#### Conversation Settings

🏚️ Before | 🏡 After
-- | --
image | image
image | image

#### Sidebars

🏚️ Before | 🏡 After
-- | --
image | image
image | image
image | image

### 🚧 Tasks

- [ ] ...

### 🏁 Checklist

- [ ] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [ ] 🖥️ Tested with Desktop client or should not be risky for it 
- [ ] 🖌️ Design was reviewed, approved or inspired by the design team
- [ ] ⛑️ Tests are included or not possible
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

File: src/components/Quote.vue [hunk 1]
```diff
@@ -247,7 +247,7 @@ export default {
 	padding: 6px 6px 6px 24px;
 	display: flex;
 	max-width: $messages-text-max-width;
-	border-radius: var(--border-radius-large);
+	border-radius: var(--border-radius-element);
```
Comment (DorraJaouad) [inline src/components/Quote.vue:250]: you are still using `--border-radius-large` in `"new-message-form__quote"`

File: src/components/MediaSettings/MediaSettings.vue [hunk 2]
```diff
@@ -711,8 +711,7 @@ export default {
 		display: flex;
```
Comment (DorraJaouad) [inline src/components/MediaSettings/MediaSettings.vue:711]: `margin: 0 auto calc(var(--default-grid-baseline) * 4);` (line above) looks better

**Review by DorraJaouad — COMMENTED** [review]
with `margin-bottom: cal(var(--border-radius-element) / 2))`  , it seems more aligned
image

File: src/components/Quote.vue (hunk 1 above)
Comment (Antreesy) [reply src/components/Quote.vue:250]: image => image

**Comment by Antreesy** [comment]
> with `margin-bottom: cal(var(--border-radius-element) / 2))` , it seems more aligned

NcSelect should also be 34px high. Let's wait for upstream fix on it first

**Review by DorraJaouad — COMMENTED** [review]
Looks good 🦅 except one thing

File: src/components/TopBar/TopBar.vue [hunk 3]
```diff
@@ -299,6 +299,7 @@ export default {
 	flex-wrap: wrap;
 	z-index: 10;
 	gap: 3px;
+	align-items: flex-start;
```
[the remark this answers is not in the document]
Comment (DorraJaouad) [reply src/components/TopBar/TopBar.vue:302]: You are right, icon size is a bit odd 👍🏻

**Review by DorraJaouad — APPROVED** [review]
🦅

File: src/components/SettingsDialog/SettingsDialog.vue [hunk 4]
```diff
@@ -390,14 +390,18 @@ export default {
...
 		height: var(--default-clickable-area);
-		padding: 10px 6px 10px 12px;
-		border: 2px solid var(--color-border-maxcontrast);
+		line-height: var(--default-clickable-area);
+		padding-inline: 12px 6px;
+		border: var(--border-width-input, 2px) solid var(--color-border-maxcontrast);
 		border-radius: var(--border-radius-large);
+		font-size: var(--default-font-size);
 		text-overflow: ellipsis;
-		opacity: 0.7;
```
Comment (DorraJaouad) [inline src/components/SettingsDialog/SettingsDialog.vue:400]: This should be kept

**Comment by ShGKme** [comment]
Great, thanks!

**Pull request merged**